libraries-server-2/src/test/java/com/baeldung/socketio/ServerLiveTest.java

```diff
@@ -0,0 +1,41 @@
+package com.baeldung.socketio;
+
+import org.junit.jupiter.api.Test;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import com.corundumstudio.socketio.Configuration;
+import com.corundumstudio.socketio.SocketIOServer;
+
+public class ServerLiveTest {
+    private static final Logger LOG = LoggerFactory.getLogger(ServerLiveTest.class);
+
+    @Test
+    void startServer() throws Exception {
```

Comment:
I've written this as a live test, but it's not really a test. It's just a convenient way to run the server code.